### PR TITLE
Telegram split

### PR DESCRIPTION
**Pull request by cescobarresi**
When configuring checks I found usefull to be able to give a multiline report to notifier 'telegram' that will result in a message per line. For example when the message has multiple urls and a preview for each would be nice to have.

This pull requests adds the notifier 'telegram-split' based on the current 'telegram' notifier. I could have added the functionality in the current 'telegram' but felt that a separate notifier was a better solution.

### CODE REVIEW

**Comment by cescobarresi** [comment]
This is intentionally compatible with the pull request https://github.com/kibitzr/kibitzr/pull/69 so that merging both wont break things.

File: docs/telegram.rst [hunk 1]
```diff
@@ -41,6 +41,28 @@ Also chat identifier can be configured for each check in ``kibitzr.yml`` using f
         notify:
           - telegram: 118860645
 
+
+Telegram Split
+--------------
+``telegram-split`` will split the report in to multiple messages before sending. The default
+``split-on`` is the new line char ``\n``. The configuration can be one of:
```
Comment (peterdemin) [inline docs/telegram.rst]: I think it's better if there will be a single `telegram` notifier with `split-on` option. If the option is not set, don't do the splitting.

**Comment by peterdemin** [comment]
> I could have added the functionality in the current 'telegram' but felt that a separate notifier was a better solution.

Funny, how I have the opposite feeling :-)
It comes from a user reading documentation and gettings confused on why is there two Telegram notifiers? Also with `split-on` parameter, it will be more DRY.

**Comment by cescobarresi** [comment]
The reasoning was that ``telegram-split`` it has a default of splitting on line break, while ``telegram`` will not spilt your message. It was a way of making the split very explicit. On the other hand having a single ``telegram`` notifier is more sane and easyer to understand for a new commer.

I don't have strong opinions on this and can modify the code to make it so that if ``split-on`` is set on ``telegram`` will split the message and if it is not set just don't split.

**Comment by peterdemin** [comment]
My preference is a single notifier with optional parameter.
Then I guess you can combine this PR with 4KB split.

**Comment by cescobarresi** [comment]
> My preference is a single notifier with optional parameter.
Will do.
> Then I guess you can combine this PR with 4KB split.
Shure but I'm not shure how to "combine" the two. Should I just merge the 4KB split in to this branch? Or there is a cleaner way?

**Comment by peterdemin** [comment]
Yeah, merge is good, but I think you’ll get a conflict so you’ll have to
resolve this manually anyway.

**Comment by cescobarresi** [comment]
With this last commit I have unified the new 'split-on' option and the automatic split of larger than 4096 char messages in to the original Telegram Notifier. Also the docs have been updated accordingly.

**Comment by peterdemin** [comment]
Thank you very much! I merged the branch, but before making release, I changed `chat-id` to `chat` for consistence with `creds`.

**Comment by cescobarresi** [comment]
Thank you for being so fast and polite with my pool requests. I have enjoyed my self doing them.

**Comment by peterdemin** [comment]
You are very welcome! Do you mind a little chat in https://gitter.im/kibitzr/Lobby?

**Pull request merged**